pom.xml

```diff
@@ -883,7 +883,7 @@
 									<repositoryId>${repository.id}</repositoryId>
 									<url>${repository.url}</url>
 									<groupId>com.enactor.chromium</groupId>
-									<artifactId>mac-binaries</artifactId>
+									<artifactId>mac-binaries-${distribution.arch}</artifactId>
```

Comment:
This isn't the way you setup a classifier in Maven.

There is a specific <classifier> tag for that purpose :)

Reply:
Will it be okay to add `<classifier>` since we using two jenkins nodes? It will override same pom.xml file from two nodes. Won't that be conflicting?